keyboards/crkbd/keymaps/default/config.h

```diff
@@ -35,10 +35,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define TAPPING_FORCE_HOLD
 #define TAPPING_TERM 100
 
-#undef RGBLED_NUM
-#define RGBLIGHT_ANIMATIONS
-#define RGBLED_NUM 27
-#define RGBLIGHT_LIMIT_VAL 120
-#define RGBLIGHT_HUE_STEP 10
-#define RGBLIGHT_SAT_STEP 17
-#define RGBLIGHT_VAL_STEP 17
+#ifdef RGBLIGHT_ENABLE
+#    undef RGBLED_NUM
+#    define RGBLIGHT_ANIMATIONS
+#    define RGBLED_NUM 27
+#    define RGBLIGHT_LIMIT_VAL 120
+#    define RGBLIGHT_HUE_STEP 10
+#    define RGBLIGHT_SAT_STEP 17
+#    define RGBLIGHT_VAL_STEP 17
```

Comment:
I don't recommend separating preprocessor directives from the `#`s. Instead, indent the whole directive, along with the `#`.

Reply:
blame clang..... :(

Reply:
Also, is there a specific reason for not doing so?

Reply:
In my own experience, it makes searching for `#define` difficult. The current coding conventions mention `Do not put whitespace between # and if` which I think should apply to defines too.

Reply:
Agreed. If `clang-format` goes against this, it should be changed.

Reply:
Will fix once I am home

Reply:
Fixed.